js/controllers/dialog.js

```diff
@@ -35,8 +35,7 @@ DialogController.prototype.show = function(callback) {
  */
 DialogController.prototype.disableEverything_ = function() {
   this.editor_.disable();
-  const inputs = document.querySelectorAll(
-      'input, select, textarea, .mdc-icon-button');
+  const inputs = document.querySelectorAll('input, textarea, .mdc-icon-button');
   for (var i = 0; i < inputs.length; i++) {
```

Comment:
I know this isn't your code, but you might want to consider

`for (const input of inputs) {`

Reply:
I might leave that one if that's ok. I try to limit myself to cleaning up the lines that I touch, otherwise i'd be here for years (: